pyscf/neo/efield.py

```diff
@@ -269,10 +268,35 @@ def __init__(self, mf):
         self.verbose = mol.verbose
         self.stdout = mol.stdout
         self._scf = mf
-
-        self.with_f1 = True # True for CNEO, False for NEO 
+        if isinstance(mf, CDFT):
+            self.with_f1 = True
+        else:
+            self.with_f1 = False
         self.max_cycle_cphf = 100
         ### default: 1e-5 is already enough
+        """
+        >>> from pyscf import neo
+        >>> from pyscf.lib import param
+        >>> from pyscf.neo.efield import Polarizability
+        >>> mol = neo.M(atom='H 0 0 0; F 0 0 0.8', basis='ccpvtz')
+        >>> mf = neo.CDFT(mol, xc='b3lyp')
+        >>> mf.kernel()
+        converged SCF energy = -100.414287028652
+        np.float64(-100.4142870286517)
+        >>> pol = neo.Polarizability(mf)        
+        >>> pol.conv_tol = 1e-5
+        >>> pol.polarizability()*param.BOHR**3
+        array([[ 4.06325330e-01, -1.24910351e-14,  6.32660972e-16],
+            [-1.24910351e-14,  4.06325330e-01,  9.83876264e-16],
+            [ 6.32660972e-16,  9.83876264e-16,  6.49102044e-01]])
+        >>> pol.conv_tol = 1e-9
+        >>> pol.polarizability()*param.BOHR**3
+        array([[ 4.06325340e-01, -2.43939181e-14,  6.35205528e-16],
+            [-2.43939181e-14,  4.06325340e-01,  9.81337957e-16],
+            [ 6.35205528e-16,  9.81337957e-16,  6.49102043e-01]])
+        Experimental uncertainty is generally 1e-3 angstrom^3. 1e-5 and 1e-9 conv_tol 
+        differ by 1e-7 angstrom^3 for the first component, so 1e-5 of conv_tol is enough.
+        """
```

Comment:
But why? Why does default pyscf use 1e-9? What is the purpose to set a loose criteria?